plugins/inputs/docker/docker_test.go

```diff
@@ -345,6 +346,12 @@ func TestDockerGatherInfo(t *testing.T) {
 			"cpu":               "cpu3",
 			"container_version": "v2.2.2",
 			"engine_host":       "absol",
+			"ENVVAR1":           "loremipsum",
+			"ENVVAR2":           "dolorsitamet",
+			"ENVVAR3":           "ubuntu:10.04",
+			"ENVVAR6":           " ",
```

Comment:
Even though this is technically correct, I think we shouldn't create a field if it contains only whitespace or an empty string.